extensions/wikia/FandomComMigration/FandomComMigration.i18n.php

```diff
@@ -0,0 +1,12 @@
+<?php
+$messages = array();
+
+$messages['en'] = array(
+	'fandom-com-migration-after' => 'Attention: We have migrated your community\'s domain to fandom.com. Find out more in <a href="https://community.wikia.com/wiki/User_blog:Brandon_Rhea/Wiki_domains_will_be_changing_from_wikia.com_to_fandom.com_in_early_2019?">this FANDOM news blog</a>.',
```

Comment:
Can we avoid the raw HTML here? I know it's temporary, so isn't a big risk security-wise, but it's still something translators have to deal with. We could change it to a wikitext external link for example (which can be parsed in JS too with `mw.message('fandom-com-migration-after').parse()`.

Reply:
:+1: updated in https://github.com/Wikia/app/pull/16126/commits/28adc8cce8cf06071207c84246172bd28376475b